lightning/src/ln/channelmanager.rs

```diff
@@ -284,6 +286,21 @@ struct PeerState {
 #[cfg(not(any(target_pointer_width = "32", target_pointer_width = "64")))]
 const ERR: () = "You need at least 32 bit pointers (well, usize, but we'll assume they're the same) for ChannelManager::latest_block_height";
 
+/// SimpleArcChannelManager is useful when you need a ChannelManager with a static lifetime, e.g.
+/// when you're using lightning-net-tokio (since tokio::spawn requires parameters with static
+/// lifetimes). Other times you can afford a reference, which is more efficient, in which case
+/// SimpleRefChannelManager is the more appropriate type. Defining these type aliases prevents
+/// issues such as overly long function definitions.
+pub type SimpleArcChannelManager<CS, M> = Arc<ChannelManager<CS, Arc<M>>>;
```

Comment:
Hmm, I think a standard user is *probably* using an InMemoryChannelKeys, so maybe replace the CS with InMemoryChannelKeys? Same below.